Client/Frontend/Intro/IntroWelcomeAndSyncViewV1.swift

```diff
@@ -0,0 +1,306 @@
+        return UpdateViewController.theme == .dark ? .black : .white
+    }
+    // Screen constants
+    private let screenHeight = UIScreen.main.bounds.size.height
+    private let screenWidth = UIScreen.main.bounds.width
+    // Views
+    private lazy var titleImageViewPage1: UIImageView = {
+        let imgView = UIImageView(image: UIImage(named: "tour-Welcome"))
+        imgView.contentMode = .center
+        imgView.clipsToBounds = true
+        return imgView
+    }()
+    private lazy var titleImageViewPage2: UIImageView = {
+        let imgView = UIImageView(image: UIImage(named: "tour-Sync"))
+        imgView.contentMode = .center
+        imgView.clipsToBounds = true
+        return imgView
+    }()
+    private lazy var titleLabel: UILabel = {
+        let label = UILabel()
+        label.text = Strings.CardTitleWelcome
+        label.textColor = fxTextThemeColour
+        label.font = UIFont.systemFont(ofSize: 32, weight: .bold)
+        label.textAlignment = .center
+        label.adjustsFontSizeToFitWidth = true
+        return label
+    }()
+    private lazy var subTitleLabelPage1: UILabel = {
+        let fontSize: CGFloat = screenWidth <= 320 ? 16 : 20
+        let label = UILabel()
+        label.text = Strings.CardTextWelcome
+        label.textColor = fxTextThemeColour
+        label.font = UIFont.systemFont(ofSize: fontSize)
+        label.textAlignment = .center
+        label.adjustsFontSizeToFitWidth = true
+        label.numberOfLines = 2
+        return label
+    }()
+    private lazy var subTitleLabelPage2: UILabel = {
+        let fontSize: CGFloat = screenWidth <= 320 ? 16 : 20
+        let label = UILabel()
+        label.text = Strings.CardTextSync
+        label.textColor = fxTextThemeColour
+        label.font = UIFont.systemFont(ofSize: fontSize)
+        label.textAlignment = .center
+        label.adjustsFontSizeToFitWidth = true
+        label.numberOfLines = 3
+        return label
+    }()
+    private var closeButton: UIButton = {
+        let closeButton = UIButton()
+        closeButton.setImage(UIImage(named: "close-large"), for: .normal)
+        if #available(iOS 13, *) {
+            closeButton.tintColor = .secondaryLabel
+        } else {
+            closeButton.tintColor = .black
+        }
+        return closeButton
+    }()
+    private lazy var signUpButton: UIButton = {
+        let button = UIButton()
+        button.accessibilityIdentifier = "signUpOnboardingButton"
+        button.titleLabel?.font = UIFont.systemFont(ofSize: 18, weight: .bold)
+        button.layer.cornerRadius = 10
+        button.backgroundColor = UIColor.Photon.Blue50
+        button.setTitle(Strings.IntroSignUpButtonTitle, for: .normal)
+        button.titleLabel?.font = UIFont.systemFont(ofSize: 18, weight: .semibold)
+        button.setTitleColor(.white, for: .normal)
+        button.titleLabel?.textAlignment = .center
+        return button
+    }()
+    private lazy var signInButton: UIButton = {
```

Comment:
This could have been done in another way (see below).
However, with all the refactoring it I was only saving few lines so decided to keep it simple. 

```
    private var topImageViews: [UIImageView] = {
        let imgView1 = UIImageView(image: UIImage(named: "tour-Welcome"))
        let imgView2 = UIImageView(image: UIImage(named: "tour-Sync"))
        let imageViews = [imgView1, imgView2]
        imageViews.forEach {
            $0.contentMode = .center
            $0.clipsToBounds = true
        }
        return imageViews
    }()
```